rules/sprint-review-retro-email/rule.md

```diff
@@ -32,12 +34,12 @@ Firstly, create a new email copying the information from the previous Sprint Rev
 ::: email-template
 |          |     |
 | -------- | --- |
-| To:      | {{Product Owner}} |
-| Cc:      | {{Sprint Review Attendees}}, {{Sprint Review Reporting Email}} |
-| Subject: | {{Product Name}} - Sprint {{X}} Review/Retro |
-::: email-content  
+| To:      | {{ PRODUCT OWNER }} |
+| Cc:      | {{ SPRINT REVIEW ATTENDEES }}, {{ PROJECT GROUP EMAIL }}, { {SPRINT REVIEW REPORTING EMAIL }} |
```

Comment:
@ricksu978 - can we add a note to the rule somewhere that says something like:

"It's important that an Email Group is setup for the project, and the Sprint Review is sent to that group, so that anyone who joins the project in future can access these reports"

Reply:
note added.